### DISCUSSION TITLE
Possible fuite mémoire avec tempel

### DISCUSSION THREAD

**Discussion by raylemon** [abougouffa/minemacs#148] [General]
Il y a une potentielle fuite mémoire avec tempel (je présume que c’est tempel, mais j’en suis pas sûr). J’ai écrit un template, mais dès que je souhaite l’appeler, Emacs se met à ramer. En regardant la consommation mémoire, je vois que Emacs monte à 10Go de RAM (99% de ma RAM totale).

Si je désactive mon template (en le renommant), Emacs consomme entre 150 à 300 Mo de mémoire. 

Pourtant mon template semble correct

```elisp
typst-ts-mode

(cours
 "#import \"cours/template.typ\": doc" n n>
 "#show: doc.with(" n>
 "title: \"" p "\"," n>
 "author: \"" p "\"," n>
 "minsize: " p"pt" n
 ")" n n
)

(chapter "#import \"cours/template.typ\": note,colors" n n)

(note "#note[" p "]" n)

(wrap "#wrapfig(" (p "alignement") "," (p "path") ")[" n p n "]")

(center) "#figure(numbering:none)[#image(\"" p "\")]"
```

**Comment by abougouffa** [comment]
Intéressant ! 

À première vue, je pense que le dernier template n'est pas correcte `(centre) ...`

Je vais essayer de reproduire ce comportement ultérieurement. Merci pour le rapport @raylemon !
